modules/formulize/class/elementrenderer.php

```diff
@@ -1042,7 +1042,7 @@ function constructElement($form_ele_id, $ele_value, $entry, $isDisabled=false, $
 			break;
 
 
-			case 'upload':
+			case 'fileUpload':
```

Comment:
This doesn't fix the problem. The fileUpload type isn't handled by this code here, it's displayed by code in the fileUploadElement.php file.